extraConfigSriov.py

```diff
@@ -127,7 +41,32 @@ def _sno_make_deploy(
         logger.info(f"Image {image} provided to load custom sriov-network-operator")
         deploy_env["SRIOV_NETWORK_OPERATOR_IMAGE"] = image
     elif build_local:
-        envs = _sno_build_local(rsh, repo_dir, client)
+        registry = start_image_registry(rsh, client)
+        project = "openshift-sriov-network-operator"
+        container_infos = [
+            ContainerInfo(
```

Comment:
Nothing to do with your PR. But it would be nice to build to device plugin as well. Maybe add a todo if you don't have cycles to do it. cc: @thom311